### PR TITLE
About link for non-en header/hamburger 

### PR DESCRIPTION
**Pull request by Erin007**
Currently, only users viewing the site in English see an "About" link in the header/hamburger. However, as our international presence has grown, it would be useful for users viewing in other languages to also have access to information about the breadth of our work. 

This PR updates the header to include an "About" link for international users, removes the professional learning link for international signed in teachers and the new international "About" link pops into the hamburger menu on mobile. 

**Signed out English desktop:** (no change from current)
signed out en desktop

**Signed out English mobile:** (no change from current)
signed out en mobile

**Signed out non-English desktop:** (additional About link)
signed out non-en desktop

**Signed out non-English mobile:** (additional About link)
signed out non-en mobile

**Teacher English desktop:** 
teacher eng desktop

**Teacher English mobile:**
teacher eng mobile

**Teacher non-English desktop:**  (additional About link, removed Professional Learning link)
teacher non-en desktop

**Teacher non-English mobile:**  (additional About link, removed Professional Learning link)
teacher non-en mobile

**Student English desktop:** (no change from current)
student en desktop

**Student English mobile:** (no change from current)
student en mobile

**Student non-English desktop:**  (additional About link)
student non-en desktop

**Student non-English mobile:** (additional About link)
student non-en mobile

### CODE REVIEW

File: lib/test/test_hamburger.rb [hunk 1]
```diff
@@ -77,6 +77,7 @@ def test_level_teacher_nonen
     assert_equal visibility[:show_signed_out_options],  Hamburger::HIDE_ALWAYS
     assert_equal visibility[:show_pegasus_options],     Hamburger::HIDE_ALWAYS
     assert_equal visibility[:show_help_options],        Hamburger::SHOW_ALWAYS
+    assert_equal visibility[:show_intl_about],        Hamburger::SHOW_MOBILE
```
Comment (breville) [inline lib/test/test_hamburger.rb]: nit: align with above options?

**Comment by Erin007** [comment]
Waiting to merge until "code.org/international/about" is ready to go. Suky's shooting for Thursday, so I'll wait a couple days rather than hide behind a flag (and add in even more logic cases!).

**Comment by Erin007** [comment]
@poorvasingal can you double check that that screenshots are what you're expecting, please. No rush!

**Comment by poorvasingal** [comment]
Wow, so many cases :) 

Comments:
- It says "Teacher English desktop: (additional About link, removed Professional Learning link)" -> but shouldn't the English version have no change from today? Your screenshot looks right. Same with mobile. So might be a copy / paste error. 
- As I'm reading through this, I'm now realizing that I don't know what the expectation is for the About page for students. If we're planning on putting PL info in there, might be weird to show to students. Let me follow up with Marina and Suky. 

Thanks for doing this and tagging me!

**Comment by poorvasingal** [comment]
Oh also - this would all be behind a flag, correct? Until page is translatable and has content?

**Comment by Erin007** [comment]
Suky said she's planning on content for international/about for Thursday, so I opted to wait a couple days to merge rather than put this behind a flag. If it ends up being way longer, I can put it behind a flag.

**Pull request merged**